content/docs/pipelines/tutorials/pipelines-tutorial.md

```diff
@@ -610,4 +602,5 @@ SDK](/docs/pipelines/sdk/sdk-overview/).
 [gsutil-mb]: https://cloud.google.com/storage/docs/gsutil/commands/mb
 [gsutil-acl-ch]: https://cloud.google.com/storage/docs/gsutil/commands/acl#ch
 
-
+[granting-changing-revoking-access]: https://cloud.google.com/iam/docs/granting-changing-revoking-access#quick_updates_using_the_gcloud_command-line_tool
```

Comment:
Sorry, this change wasn't called out as well as I would have liked in my previous review. Would you please update this link to be the following? 

https://cloud.google.com/iam/docs/granting-changing-revoking-access#using_the

This will take users to the instructions for updating IAM settings in the Cloud Console instead of the Cloud Shell.

Reply:
Got it, updated.